lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/expression/ConnectorExpressionPatterns.java

```diff
@@ -74,6 +74,11 @@ private ConnectorExpressionPatterns() {}
         });
     }
 
+    public static Property<Call, ?, List<ConnectorExpression>> arguments()
```

Comment:
remove from "Translate COALESCE to connector expressions" commit

Reply:
BTW we may or may not need this. Capturing Call and getting it's arguments is also fine.

Reply:
Removed to capture as Call.